src/main/java/com/ensao/gi5/lint/rules/ConstantClassNameRule.java

```diff
@@ -0,0 +1,42 @@
+package com.ensao.gi5.lint.rules;
+
+import com.ensao.gi5.lint.constantes.Constantes;
+import com.ensao.gi5.lint.rules.violations.Violation;
+import com.ensao.gi5.lint.wrapper.CompilationUnitWrapper;
+import com.github.javaparser.ast.body.FieldDeclaration;
+
+public class ConstantClassNameRule extends Rule {
+    public ConstantClassNameRule() {
+        super(Constantes.LINT_REG_005, Level.MEDIUM);
+    }
+
+    @Override
+    public void apply(CompilationUnitWrapper compilationUnit) {
+            compilationUnit.getCompilationUnit().findAll(FieldDeclaration.class).forEach(fieldDeclaration -> {
+                if (fieldDeclaration.isFinal() && fieldDeclaration.isStatic()) {
+                    String name = fieldDeclaration.getVariables().get(0).getNameAsString();
```

Comment:
get(0) is to be generalized : 
```java
public static final String FOO = "foo", BAR = "BAR" ; 
``` 
The rule will be applied on the first element only.